src/utils/parsers/common.ts

```diff
@@ -306,7 +306,20 @@ export abstract class BaseParser {
         return results
     }
 
-    parseFile(_file: usage.File) {
-        return []
+    parseFile(_file: usage.File): FileMatch[] {
```

Comment:
could remove the `_` from the `file` param, I'd just added that so it wouldn't complain about an unused param